tools/api-tests/src/services/user.ts

```diff
@@ -20,7 +20,7 @@ const signUser = (id: string): string => {
   }
 
   // TODO: This should come out of config
```

Comment:
It's very difficult to get access to the application config when running in a docker environment as it can't use parts of the filesystem outside of it's build context - Maybe we could add a build step that copies the config into the `api-test` context but that's another unit of work

Reply:
the build context is for building the image, but is the container able to get `../../something` mounted as a volume? You can in general mount arbitrary host paths so should be possible.